style.css

```diff
@@ -315,9 +343,12 @@ div.multi-post-story > :last-child {
 }
 
 div.see {
-  width: 32.7rem;
-  margin-left: auto;
-  margin-right: auto;
+  width: 90%;
+
+  /* margin-left: auto; */
+
+  /* margin-right: auto; */
```

Comment:
- [ ] I kindly suggest to avoid leaving commented code in your code base. This is because Leaving commented-out code in source code is bad practice, as it takes up space, causes confusion, and leads to maintenance issues if it's not removed 😞.